test/benchmarks/CMakeLists.txt

```diff
@@ -62,9 +62,6 @@ if(WIN32)
     target_link_libraries(benchmark_zlib shlwapi)
 endif()
 
-add_test(NAME benchmark_zlib
-    COMMAND ${CMAKE_CROSSCOMPILING_EMULATOR} $<TARGET_FILE:benchmark_zlib>)
-
```

Comment:
Please tell me why you removed the ``benchmark_zlib`` cmake-test?

Reply:
It is in the PR description.

```
Also added a small unrelated "fix" for an annoyance while doing this, to stop running
gbench as part of gtest. It can take minutes to complete benchmarking, but it hides
the results so it is not so useful to run as a gtest.
```

Reply:
`benchmark_zlib` can of course still be run directly from the command line.

Reply:
Thanks for your answer.